app/controllers/concerns/form_attachment_create.rb

```diff
@@ -11,7 +11,10 @@ def create
     form_attachment_model = self.class::FORM_ATTACHMENT_MODEL
     form_attachment = form_attachment_model.new
     namespace = form_attachment_model.to_s.underscore.split('/').last
-    form_attachment.set_file_data!(params[namespace][:file_data])
+
+    form_attachment_params = params.require(namespace).permit(:file_data)
```

Comment:
Might something like `attached_form` read better? There's only one param so params doesn't read well to me. 

```
attached_form = params.require(namespace).permit(:file_data)
form_attachment.set_file_data!(attached_form)
```

Maybe even `file_data` to "hoist" that param name up to the top level

```
file_data = params.require(namespace).permit(:file_data)
form_attachment.set_file_data!(file_data)
```

Reply:
👍